src/derived/grid8.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
is it really worth having an entire core API component for grid8? Its so so simple that it could just be left to the consumer to create a reusable component.